react/ManualPrice.tsx

```diff
@@ -54,8 +54,11 @@ const ManualPrice: FunctionComponent = () => {
   }
 
   const revertToOriginalPrice = () => {
-    onSetManualPrice(item.price!, itemIndex)
-    setManualPrice(item.price)
+    if (item.price) {
```

Comment:
shouldn't this check only for nullish values? if the price is `0` it wouldn't work
```suggestion
    if (item.price != null) {
```

Reply:
hm, you're right!